docs/rpc.md

```diff
@@ -31,41 +31,6 @@ which would yield the response:
 }
 ```
 
-### Supported methods
-
-| Method  | Implemented |
-| ------- | ------------|
-| `getaccountstate` | Yes |
-| `getapplicationlog` | Yes |
-| `getassetstate` | Yes |
-| `getbestblockhash` | Yes |
-| `getblock` | Yes |
-| `getblockcount` | Yes |
-| `getblockhash` | Yes |
-| `getblockheader` | Yes |
-| `getblocksysfee` | Yes |
-| `getclaimable` | Yes |
-| `getconnectioncount` | Yes |
-| `getcontractstate` | Yes |
-| `getnep5balances` | Yes |
-| `getnep5transfers` | Yes |
-| `getpeers` | Yes |
-| `getrawmempool` | Yes |
-| `getrawtransaction` | Yes |
-| `getstorage` | Yes |
-| `gettransactionheight` | Yes |
-| `gettxout` | Yes |
-| `getunclaimed` | No (#712) |
```

Comment:
Don't drop it! Just drop the "Implemented" column, make it a list or something. But we need to have a full list of implemented methods in the documentation.